pwa/src/templates/componentResult/ComponentResultsTemplate.module.css

```diff
@@ -0,0 +1,10 @@
+.contentRow > :last-child {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.componentsGrid {
+  display: grid;
+  grid-gap: var(--nlportal-space-block-md);
+  grid-template-columns: 1fr 1fr;
+}
```

Comment:
This css rules themselves are fine. However, its unclear to what sub-component they're referring. 

e.g.: the `.contentRow` is targeting the table, but that's not clear within the context of this css.

A solution would be to add some comments to this css file, like so:

```
/*  Table */
... all table rules

/* Cards */
... all cards rules

/* Layers */
... all layer rules
```